pycaching/geocaching.py

```diff
@@ -356,6 +363,51 @@ def search_quick(self, area, *, strict=False, zoom=None):
 
     # add some shortcuts ------------------------------------------------------
 
+    def search_rect(self, rect: Rectangle, *, per_query: int = 50, sortby: str = 'datelastvisited', origin=None):
+        """
+        Return a generator of caches in given Rectange area.
+
+        :param rect: Search area.
+        :param per_query: Number of caches requested in single query.
+        :param sortby: Order cached by given criterion.
+        :param origin: Origin point for search by distance.
+        """
+        assert sortby in {
+            'containersize', 'datelastvisited', 'difficulty', 'distance', 'favoritepoint',
```

Comment:
I think we should use an enumeration here.

Reply:
done, I have defined enum `SortOrder`